### PR TITLE
[6.3]Added CLI stubs for oscap tailoring files

### PR DESCRIPTION
**Pull request by ntkathole**

### CODE REVIEW

File: tests/foreman/cli/test_oscap_tailoringfiles.py [hunk 1]
```diff
@@ -0,0 +1,348 @@
...
+    @stubbed()
+    @tier1
+    def test_positive_create_with_space(self):
+        """Create tailoring files with space in name
+
+        :id: c98ef4e7-41c5-4a8b-8a0b-8d53100b75a8
+
+        :steps:
+
+            1. Create valid tailoring file with valid parameter
```
Comment (san7ket) [inline tests/foreman/cli/test_oscap_tailoringfiles.py]: `1. Create valid tailoring file with space in name.`

No need of 2 steps

File: tests/foreman/cli/test_oscap_tailoringfiles.py [hunk 2]
```diff
@@ -0,0 +1,348 @@
...
+
+        :expectedresults: ARF report should be sent to satellite
+                         reflecting the changes done via tailoring files
+
+        :CaseImportance: Critical
+        """
+
+    @run_only_on('sat')
+    @stubbed()
+    @tier3
```
Comment (san7ket) [inline tests/foreman/cli/test_oscap_tailoringfiles.py]: tier4

File: tests/foreman/cli/test_oscap_tailoringfiles.py [hunk 3]
```diff
@@ -0,0 +1,348 @@
...
+        :CaseAutomation: notautomated
+
+        :expectedresults: The tailoring file should be downloaded
+
+        :CaseImportance: Critical
+        """
+
+    @run_only_on('sat')
+    @stubbed()
+    @tier1
```
Comment (san7ket) [inline tests/foreman/cli/test_oscap_tailoringfiles.py]: tier2

File: tests/foreman/cli/test_oscap_tailoringfiles.py [hunk 4]
```diff
@@ -0,0 +1,348 @@
...
+        :CaseAutomation: notautomated
+
+        :expectedresults: Association should give some warning
+
+        :CaseImportance: Critical
+        """
+
+    @run_only_on('sat')
+    @stubbed()
+    @tier1
```
Comment (san7ket) [inline tests/foreman/cli/test_oscap_tailoringfiles.py]: tier2

File: tests/foreman/cli/test_oscap_tailoringfiles.py [hunk 5]
```diff
@@ -0,0 +1,348 @@
...
+        :CaseAutomation: notautomated
+
+        :expectedresults: Association should be successful
+
+        :CaseImportance: Critical
+        """
+
+    @run_only_on('sat')
+    @stubbed()
+    @tier1
```
Comment (san7ket) [inline tests/foreman/cli/test_oscap_tailoringfiles.py]: tier2

**Comment by ntkathole** [comment]
@san7ket thanks for review...updated tiers and nitpick

**Comment by sghai** [comment]
ACK

**Pull request merged**